features/support/capybara.rb

```diff
@@ -7,7 +7,7 @@
 #  Capybara::Selenium::Driver.new(app, :browser => :firefox)
 #end
 
-DEFAULT_JS_DRIVER = :headless_chrome
+DEFAULT_JS_DRIVER = :headless_firefox
 # in case firefox is needed!
```

Comment:
Perhaps we could invert this comment now to "in case chrome is needed"

Reply:
Or just remove the comment 😄